### DISCUSSION TITLE
Lightspeed on ethernet

### DISCUSSION THREAD

**Discussion by funkymokey** [3kh0/ext-remover#880] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Has anyone seen a challenge bigger than this
![Screenshot (4)](https://github.com/3kh0/ext-remover/assets/120732996/e4387c64-35ed-4f0d-9f4f-086d5d00d87e)

**Comment by rwg100** [comment, marked as answer]
i have no fucking clue what you are saying here

**Comment by AshtonDavies** [comment]
Looks like Lightspeed uses more than twice the data of any other application you have installed. LOL.
